scripts/gan/srgan/README.md

```diff
@@ -0,0 +1,32 @@
+## Reproducing SRGAN experiments
+
+![images](pred.png "images ")
+
+**Download DIV2K dataset**
+```bash
+bash ./download_dataset.sh DIV2K_train_HR
```

Comment:
please use a python script instead of bash

Reply:
Yes, This is a mistake of the README file.